src/index.html

```diff
@@ -3,12 +3,60 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-content">
+        <img src="../src/images/logo.png" alt="logo">
+        <nav class="nav">
+          <ul class="nav__items">
+            <li class="nav__item">
+              <a href="#" class="nav__item-link is-active">APPLE</a>
```

Comment:
You have "text-transform: uppercase;" in css
So you don't need to write text in uppercase in html
"APPLE" -> 'Apple'
The same in other places